src/password/scripts/directives.js

```diff
@@ -66,13 +63,15 @@
     module.directive('passwordChangeForm', [
         '$route',
         'passwordFactory',
-        function ($route, passwordFactory) {
+        'catchErrors',
+        function ($route, passwordFactory, catchErrors) {
             return {
                 restrict: 'A',
                 scope: true,
                 templateUrl: 'templates/user_management/password/change_form.html',
                 link: function (scope, element, attrs) {
                     scope.data = {};
+                    scope.errors = {};
```

Comment:
You no longer need to assign `scope.errors = {}` on line 116, in `changePassword()`